app/scripts/controllers/reviewRegistration.js

```diff
@@ -280,4 +280,141 @@ angular.module('confRegistrationWebApp')
     $scope.hasPendingCheckPayment = function(payments){
       return _.some(payments, { paymentType: 'CHECK', status: 'PENDING' });
     };
+
+    ////// START EVENT-433 //////
+
+    // Get spouse registration
+    // End point: getSpouseRegistration = /conferences/conferenceId/registrations/spouse
+    ///// TEST DATA change end point url below to: /conferences/conferenceId/registrations/current /////
+    $http.get('conferences/' + conference.id + '/registrations/spouse')
+      .then(function(response) {
+        if (response.status === 200) {
+          $scope.spouseRegistration = response.data;
+
+          ///// TEST DATA /////
+          // $scope.spouseRegistration.completed = true;
+          // $scope.spouseRegistration.registrants[0].email = 'janeworner3@hotmail.com';
+          // $scope.spouseRegistration.registrants[0].firstName = 'Jane';
+          // $scope.spouseRegistration.registrants[0].lastName = 'Worner3';
+          ///// TEST DATA ////
+
+        }
+        else {
+          $scope.spouseRegistration = {};
+          console.log('No current and/or spouse registration found for user with ID.');
+        }
+      }).catch(function(response) {
+      $scope.spouseRegistration = {};
+      console.log('No current and/or spouse registration found for user with ID. Status = ' + response.status);
+      console.log(response.data);
+    })
+      .finally(function() {
+        checkSpouseRegistration();
+      });
+
+    //Parameters needed to retrieve all current registrants for conference.id
+    var queryParameters = {
+      block: [],
+      //page: 1,
+      //limit: 20,
+      orderBy: 'last_name',
+      order: 'ASC',
+      filter: '',
+      filterPayment: '',
+      filterRegType: '',
+      includeCheckedin: 'yes',
+      includeWithdrawn: 'yes',
+      includeIncomplete: 'yes'
+    };
+
+    //Get all current registrants based on conference.id and queryParameters
+    //then check if spouse is registered and display 'Spouse Registration' section if needed.
+    var checkSpouseRegistration = function() { // jshint ignore:line
+      $http.get('conferences/' + conference.id + '/registrations', {params: queryParameters})
```

Comment:
@dougworner Why would we need to check the entire list of registrants against the spouse registration? We only need to know if any of the registrants on the current registration are already on the spouse registration. That can be accomplished with a query like this:

```js
!_.isEmpty(_.intersection(_.map($scope.currentRegistration.registrants, 'email'), _.map($scope.spouseRegistration.registrants, 'email')));
```

With the merge registrations button, it should be shown in place of the confirm button. When the user clicks it, it should merge the registrations and then redirect the user to the spouse's registration page because it was the winner of the merge.

P.S. You can create code blocks by surrounding them with triple backticks. I edited your comment to use them, which makes it more readable. You can read more about code highlighting in GitHub Markdown [here](https://help.github.com/articles/creating-and-highlighting-code-blocks/).